dev/com.ibm.ws.jaxrs.2.1_fat_executor/test-applications/jaxrsapp/src/web/jaxrstest/JAXRSExecutorTestServlet.java

```diff
@@ -159,7 +169,13 @@ public void testCompletionStageRxInvokerAsynchronousFunction(HttpServletRequest
         } else {
             assertTrue(resultString, executionThreadName.startsWith("Default Executor-thread-"));
         }
-        assertTrue(resultString, results[2] instanceof NamingException);
+        // In EE10 the thread holds the proper context so the java:comp lookup is successful.
+        if (ee10) {
+            assertTrue("Not expected ManagedScheduledExecutorService", (results[2] instanceof ManagedScheduledExecutorService));
```

Comment:
I am quite confused on how this doesn't fail with concurrent-3.0 feature being enabled.  The class should not even be loadable.

Reply:
I believe that the class was loaded because it was one of our overrides.  Anyhow, the class has been changed per your earlier suggestion.

Reply:
My point is how does this servlet access ManagedScheduledExecutorService when not running with concurrent feature in the server.xml.  It should get a NoClassDefFoundError and the jndi lookup should return null I believe since it shouldn't havea  DefaultManaged executor when concurrent-3.0 feature is not enabled.  That was the point of my comment, not the RESTEasy class in the previous comment.

Reply:
After discussion we decided to remove the repeat of the test without the concurrency feature and work that under a separate issue/pr